src/AVFoundation/AVAssetDownloadStorageManagementPolicy.cs

```diff
@@ -18,7 +20,7 @@ public partial class AVMutableAssetDownloadStorageManagementPolicy {
 		
 		public override AVAssetDownloadedAssetEvictionPriority Priority {
 			get { return AVAssetDownloadedAssetEvictionPriorityExtensions.GetValue (_Priority); }
-			set { _Priority = value.GetConstant (); }
+			set { _Priority = value.GetConstant ()!; }
```

Comment:
I would throw an exception since if GetConstant returns null it means that AVAssetDownloadedAssetEvictionPriority is out of range. Setters can throw exceptions.